.storybook/preview.js

```diff
@@ -14,7 +14,9 @@ Dependencies.setDependencies(storybookTestDependencies);
 export const decorators = [
     (Story) => (
         <RenderStateRoot>
-            <Story />
+            <div class="framework-perseus">
```

Comment:
This helps with editor styles because many of them are defined as descendants of `.framework-perseus`

Reply:
I don't see a big problem with this. My instinct though: we'd like to move to scoped CSS so we avoid the conflicts that plagued us during The Big Release. By putting this here, people who write stories might be blissfully unaware that this class is affecting their components - out of sight, out of mind.

However if we wrapped individual stories with this it becomes very obvious which components are banking on this class and will help us to find these components and scope their CSS.

So I don't consider it a blocker, but I might advocate for putting this in individual stories over the story wrapper.

Reply:
I like that idea.  So in favour of moving these global CSS rules to component styling on shared components.